src/pybel/io/mongodb.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+"""An exporter from PyBEL graphs to a local Mongo database."""
+
+import logging
+from copy import deepcopy
+from typing import Any, List, Mapping, Optional, Tuple
+
+from pymongo import MongoClient
+from pymongo.collection import Collection
+
+from pybel import BELGraph, to_sbel
+from pybel.dsl import Entity, Variant
+from pybel.schema import is_valid_edge, is_valid_node
+
+__all__ = [
+    '_entity_to_dict',
+    '_rm_mongo_keys',
+    'to_mongodb',
+    'find_nodes',
+    'get_edges_from_node',
+    'get_edges_from_criteria'
+]
+
+logger = logging.getLogger(__name__)
+
+
+def _entity_to_dict(entity: Entity) -> Mapping[str, Any]:
+    """Input a pybel Entity and return a dict representing it."""
+    new_node = dict(entity)
+    if new_node['function'] in ['Complex', 'Composite']:
+        new_node['members'] = list(map(dict, new_node['members']))
+    return new_node
+
+
+def to_mongodb(
+    graph: BELGraph,
+    db_name: str, collection_name: str,
```

Comment:
would you please choose a default `db_name` and `collection_name` in case the user just wants reasonable defaults?

Save these as constants at the top of the file, then make these two arguments`Optional[str] = None`. Inside the function, you can do the same none check as with the client